src/platform/packages/shared/kbn-scout/src/config/utils/save_scout_test_config.test.ts

```diff
@@ -34,6 +35,7 @@ const testServersConfig = {
     password: 'changeme',
   },
   serverless: true,
+  projectType: 'oblt' as ServerlessProjectType,
```

Comment:
Out of curiosity, is there any specific reason we default the `projectType` to the Observability project type?

Reply:
no, just has to be one of typed ones.